app/views/arf_reports/welcome.html.erb

```diff
@@ -9,6 +9,6 @@
     <%= _("You don't seem to have any ARF report. ARF report is a summary of a single scan occurrence on a particular host for a given Compliance Policy.") %></br>
   </p>
   <div class="blank-slate-pf-main-action">
-    <%= link_to _('Documentation'), documentation_url("4.4ARFReports", :root_url => "https://www.theforeman.org/plugins/foreman_openscap/0.8/index.html#"), :rel => 'external', :class => 'btn btn-primary btn-lg'  %>
+    <%= link_to _('Documentation'), documentation_url("Monitoring_Compliance_security-compliance", :root_url => scap_doc_url), :rel => 'external', :class => 'btn btn-primary btn-lg'  %>
```

Comment:
Shouldn't this use `scap_doc_url`?
```suggestion
    <%= link_to _('Documentation'), scap_doc_url("Monitoring_Compliance_security-compliance"), :rel => 'external', :class => 'btn btn-primary btn-lg'  %>
```

For a moment I thought about using `scap_doc_link` but that looks to be not flexible enough.